### ISSUE TITLE
arXiv 2021 | Training Larger Networks for Deep Reinforcement Learning

### ISSUE THREAD

**Issue by ChufanSuki** [ChufanSuki/read-paper-and-code#151]
https://arxiv.org/abs/2102.07920

**Comment by ChufanSuki** [comment]
It has been reported in some studies that deep RL agents experience instability while training with larger networks.  This is contrary to our intuition considering the recent progress on solving computer vision tasks such ViT: larger and more complex network architectures have proven to achieve better performance.

Sutton identifies a deadly triad of _function approximation_, _bootstrapping_, and _off-policy_ learning. When these three properties are combined, learning can be unstable, and potentially diverge with the value estimates becoming unbounded. Some prior works have challenged to mitigate this problem including
- target network
- double Q-learning
- n-step learning

This paper try to mitigate this problem related to function approximation. Previous work(e.g. make MLP/CNN network larger) concluded the larger networks tend to perform better, but also become unstable and prone to diverge more. For on-policy methods, too small or large networks can cause significant drop in performance of the policy.

**Comment by ChufanSuki** [comment]
### Methods

1.  Decoupling Representation Learning from RL: OFENet+auxiliary task $\rightarrow$ decoupling unsupervised pretraining from downstream task
2. Distributed Training: Ape-X $\rightarrow$ mitigate overfitting and rank collapse issues of Q-networks
3. Network Architectures: DenseNet $\rightarrow$ improved flow of information and gradients throughout the network
